src/dashboard/pages/dashboard.py

```diff
@@ -0,0 +1,74 @@
+from collections import OrderedDict
+
+import dash
+from dash import dcc, html
+import plotly.express as px
+
+from dashboard.components.icon import icon
+
+dash.register_page(__name__, path_template="/dashboard/<dashboard_id>")
+
+# Ordered dict containing the name of the button and its corresponding icon name
+# Find the icon names here https://fonts.google.com/icons?icon.set=Material+Symbols
+buttons = OrderedDict(
+    [("Presentation view", "present_to_all"), ("Share", "share"), ("Add graph", "add")]
+)
+
+
+def add_buttons(button_names: OrderedDict[str, str]) -> list[html.Button]:
```

Comment:
`add` suggests that buttons is added to something. Use a more suitable verb here.